docs/docsite/keyword_desc.yml

```
@@ -17,6 +18,11 @@ check_mode: |
 
     .. seealso:: :ref:`check_mode_dry`
 
+collections: |
+    List of collection namespaces to search for modules.
```

```suggestion
    List of collection namespaces to search for modules and roles.
```
Maybe this should also note that tasks in a role do not inherit the playbook keyword (or that the collections keyword needs to be used in meta/main.yml to apply to tasks within a role?).

---

Good point. I will add a note explaining this.